README.md

```diff
@@ -73,12 +73,67 @@ gulp.task('default', gulp.series('clean', 'all'));
 
 ## Incremental Builds
 
-We recommend these plugins:
+You can filter out unchanged files between runs of a task using 
+`gulp.src` `since` option and `gulp.lastRun`:
```

Comment:
Needs a possessive. I'd suggest "using the `gulp.src` function's `since` option".